app/experimenter/kinto/tests/test_tasks.py

```diff
@@ -25,6 +30,18 @@ def test_push_experiment_to_kinto_sends_experiment_data(self):
 
         data = ExperimentRapidRecipeSerializer(self.experiment).data
 
+        self.assertTrue(
+            ExperimentBucketRange.objects.filter(experiment=self.experiment).exists()
+        )
+
+        bucketConfig = data["arguments"]["bucketConfig"]["count"]
+
+        designPreset = NIMBUS_DATA["ExperimentDesignPresets"]["empty_aa"]["preset"][
+            "arguments"
+        ]["bucketConfig"]["count"]
+
+        self.assertEqual(bucketConfig, designPreset)
```

Comment:
this seems kinda weird to be only checking count.... but total, namespace, and start aren't part of the designPresets. and the randomization differs where it's "userId" in the designPreset but experimenter defaults it to "normandy_id". Not sure whether this is intentional

Reply:
OH snap! yes it should be what's in the design preset!  

https://github.com/mozilla/nimbus-shared/commit/336d51620c1ac6d44417f7c4a3dff83d34f6e706

So we gotta update the serializer to use that, and check it here in the test.  Good catch.